src/main/kotlin/calculator/InputParser.kt

```diff
@@ -6,18 +6,21 @@ import calculator.exceptions.InvalidCharacterException
 import calculator.exceptions.NotEnoughElementsException
 
 object InputParser {
+    private val charactersRegex = Regex(CHARACTERS_PATTERN)
+    private val splitRegex = Regex(SPLIT_PATTERN)
```

Comment:
What I meant is that `Regex` is effectively a constant object. So even though you can’t add the `const` modifier, you can still follow the constant naming convention. Please refer to the Kotlin style guide below:
https://developer.android.com/kotlin/style-guide#constant_names

```suggestion
    private val CHARACTERS_REGEX: Regex = Regex("[^0-9+\\-*/\\s]")
    private val SPLIT_REGEX: Regex = Regex("\\s")
```